packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -1,33 +1,283 @@
 <template>
     <div
-        :class="$style['c-mediaElement']"
-        data-test-id="mediaElement-component">
-        {{ componentName }}
+        data-test-id="mediaElement-component"
+        :class="[
+            $style['c-mediaElement'],
+            (stacked ? $style['c-mediaElement--stack'] : ''),
```

Comment:
Could you do ```{ [$style['c-mediaElement--stack'']]: stacked }```

Reply:
What I did originally and ash suggested to change

Reply:
Ash suggested that change, I had it like this originally. @ashleynolan could you explain why the suggested change is better?

Reply:
I think this way is a little cleaner (less nested brackets and the conditional at the front of the statement) – equally, if other people don't like it and prefer the other way happy to discuss (shouldn't stop this PR though)

Reply:
I'd personally favour `[$style['c-mediaElement--stack'']]: stacked` over `(stacked ? $style['c-mediaElement--stack'] : '')` 🙂

Reply:
I will leave as is for now, but can we please agree as a team what choice we want to make, my preference is the way @FayeCarter suggests.

Reply:
Yep – I'll setup a github discussion so we can agree on an approach going forward 👍